tutorial/consuming_packages/cross_building_with_conan.rst

```diff
@@ -0,0 +1,217 @@
+.. _consuming_packages_cross_building_with_conan:
+
+How to cross-compile your applications using Conan: host and build contexts
+===========================================================================
+
+.. important::
+
+    In this example, we will retrieve Conan packages from a Conan repository with
+    packages compatible with Conan 2.0. To run this example successfully you should add this
+    remote to your Conan configuration (if did not already do it) doing:
+    ``conan remote add conanv2 https://conanv2beta.jfrog.io/artifactory/api/conan/conan --index 0``
+
+
+Please, first clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/tutorial/consuming_packages/cross_building
+
+
+In the previous examples, we learned how to use a *conanfile.py* or *conanfile.txt* to
+build an application that compresses strings using the *Zlib* and *CMake* Conan packages.
+Also, we explained that you can set information like the operating system, compiler or
+build configuration in a file called the Conan profile. You can use that profile as an
+argument (``--profile``) to invoke the :command:`conan install`. We also explained that
+not specifying that profile is equivalent to using the ``--profile=default`` argument.
+
+For all those examples, we used the same platform for building and running the
+application. But what if you want to build the application in your machine running Ubuntu
+Linux, but you would like that application to run on other platform like, for example, a
+Raspberry Pi? Conan can model that case using two different profiles, one for the
+machine that **builds** the application (Ubuntu Linux) and another for the machine that
+**runs** the application (Raspberry Pi). We will explain this "two profiles" approach in
+the next section.
+
+Conan two profiles model: build and host profiles
+-------------------------------------------------
+
+Although for every conan command you can specify only one ``--profile`` argument, Conan
+will internally use two profiles. One for the machine that **builds** the binaries (called
+the **build** profile) and another for the machine that **runs** those binaries (called
+the **host** profile). Calling this command:
+
+.. code-block:: bash
+
+    $ conan install . --build=missing --profile=someprofile
+
+Is equivalent to:
+
+.. code-block:: bash
+
+    $ conan install . --build=missing --profile:host=someprofile --profile:build=someprofile
+
```

Comment:
I don't think that is correct:

 - `--profile` is equivalent to `--profile:host` 
 - If you don't specify `--profile:build` the applied build profile `is not` `someprofile`, is `default`.